File: 단계별로 풀어보기/조건문/Python/2525.py
# 2525
# 오븐 시계
# 브론즈4

import sys

hour, minute = map(int,sys.stdin.readline().split())
time = int(sys.stdin.readline())

if (minute + time < 60):
    minute = minute + time
else:
    hour += (minute + time)//60
    minute = (minute + time)%60
    
if (hour>=24):
    hour = hour - 24    
print(hour, minute)

File: 단계별로 풀어보기/조건문/C++/2525.cpp
// 2525
// 오븐 시계
// 브론즈4

#include <iostream>
using namespace std;

int main()
{
    int hour, minute, time = 0;

    cin>>hour>>minute>>time;
    
    if (minute + time < 60){
        minute = minute + time;
    }    
    else{
        hour += (minute + time)/60;
        minute = (minute + time)%60;
    }   
    
    if (hour>=24){
        hour = hour - 24;
    }        
    cout<<hour<<" "<<minute;
}